manim/renderer/cairo_renderer.py

```diff
@@ -30,7 +31,6 @@ def __init__(
         # All of the following are set to EITHER the value passed via kwargs,
         # OR the value stored in the global config dict at the time of
         # _instance construction_.
-        self.file_writer = None
```

Comment:
We should decide whether class attributes should in general be initialized during `__init__`, even if they are only used or actually assigned later on. Removing this is fine, for now.